src/main/java/soconnect/model/person/UniquePersonList.java

```diff
@@ -131,6 +131,29 @@ public void changeRelevantPersonTag(Tag oldTag, Tag newTag) {
         }
     }
 
+    /**
+     * Removes the tag in every contact.
+     *
+     * @param tag The tag to be removed.
+     */
+    public void removeRelevantPersonTag(Tag tag) {
+        for (int i = 0; i < internalList.size(); i++) {
+            Person oldPerson = internalList.get(i);
+            List<Tag> personTags = new ArrayList<>(oldPerson.getTags());
+            if (!personTags.contains(tag)) {
+            } else {
+                personTags.remove(tag);
+                Set<Tag> updatedTags = new HashSet<>(personTags);
+                Person updatedPerson = new Person(oldPerson.getName(),
+                    oldPerson.getPhone(),
+                    oldPerson.getEmail(),
+                    oldPerson.getAddress(),
+                    updatedTags);
+                internalList.set(i, updatedPerson);
```

Comment:
Okay, then this should be `removeRelevantTodoTag(Tag tag)` right since it is removing the deleted tags in `Todo`s.

Then remember to update the tag delete command, make sure it calls this `removeRelevantTodoTag` when a `tag` is deleted.